### ISSUE TITLE
3.5 Tooltip title attribute not accessible.

### ISSUE THREAD

**Issue by dominikwilkowski** [WestpacCXTeam/GUI-source#80]
[3.5](http://maxdesign.com.au/jobs/GUI2.0-accessibility-model-report/#sec3-5)
